src/pages/workspace/card/WorkspaceCardVBAWithECardView.tsx

```diff
@@ -2,28 +2,35 @@ import React from 'react';
 import {View} from 'react-native';
 import * as Expensicons from '@components/Icon/Expensicons';
 import * as Illustrations from '@components/Icon/Illustrations';
+import type {MenuItemWithLink} from '@components/MenuItemList';
 import Section from '@components/Section';
 import Text from '@components/Text';
 import UnorderedList from '@components/UnorderedList';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Link from '@userActions/Link';
 
-const propTypes = {
-    ...withLocalizePropTypes,
+type MenuLinks = {
+    ISSUE_AND_MANAGE_CARDS: string;
+    RECONCILE_CARDS: string;
+    SETTLEMENT_FREQUENCY: string;
 };
 
-const MENU_LINKS = {
+type MenuItems = MenuItemWithLink[];
+
+const MENU_LINKS: MenuLinks = {
     ISSUE_AND_MANAGE_CARDS: 'domain_companycards',
     RECONCILE_CARDS: encodeURI('domain_companycards?param={"section":"cardReconciliation"}'),
     SETTLEMENT_FREQUENCY: encodeURI('domain_companycards?param={"section":"configureSettings"}'),
 };
```

Comment:
```suggestion
const MENU_LINKS = {
    ISSUE_AND_MANAGE_CARDS: 'domain_companycards',
    RECONCILE_CARDS: encodeURI('domain_companycards?param={"section":"cardReconciliation"}'),
    SETTLEMENT_FREQUENCY: encodeURI('domain_companycards?param={"section":"configureSettings"}'),
} as const;
```

This way is more simple and fulfil the same purpose, also remove `MenuLinks` :)